packer/scripts/macos/macos-agentsetup.sh

```diff
@@ -28,6 +28,8 @@ yes | sudo port install openjdk17-temurin
 yes | sudo port install openjdk21-temurin
 yes | sudo port install jenv
 echo 'eval "$(jenv init -)"' >> ~/.bash_profile && source ~/.bash_profile
+jenv enable-plugin export
```

Comment:
Thanks @jordarlu  so without this`jenv enable-plugin export` it is now not able to add the java home env ?

Reply:
you are right, @prudhvigodithi , without enabling this plugin, the jenv is not setting up any system enviroment; hence, the $JAVA_HOME is null at the moment.